examples/common/scripts/vtadmin-down.sh

```diff
@@ -2,8 +2,17 @@
 
 source "$(dirname "${BASH_SOURCE[0]:-$0}")/../env.sh"
 
-echo "Stopping vtadmin-web..."
-kill -9 "$(cat "$VTDATAROOT/tmp/vtadmin-web.pid")"
+function stop_vtadmin() {
+  local name="$1"
+  local file="$2"
 
```

Comment:
We should check that these values were actually passed:
```suggestion

  if [[ -z ${1} || -z ${2} ]]; then
    fail "A binary name and PID file must be specified when attempting to shutdown vtadmin"
  fi
```

Reply:
Yes but no.  This function is only used internally in the file, so IMHO not worth my time, but feel free to do it yourself if you think it is worth your time.